README.txt

```diff
@@ -0,0 +1,81 @@
+Raleway Variable Font
```

Comment:
as i said you downloaded it from somewhere, so i exposed two solutions find the one that feat you the most. readme documents just documentation files. so either you delete it because it's of no use to you or you put everything related to this into a sub folder in src/  up to you to decide.

Reply:
Looks like it was for the new font-family, I put it in a file !